### PR TITLE
Update kube dependencies to v1.19.3

### PR DESCRIPTION
**Pull request by alvaroaleman**
This PR updates the kube and controller-runtime dependencies to their v1.19 version.

~WIP because there is no controller-runtime release yet and because there is an unresolved issue with an ingress conversion.~

Lets just merge this even without using a released controller-runtime version as that is happening very slow this time around. We can bring in a released version of it once it exists.

### CODE REVIEW

File: prow/plank/reconciler.go [hunk 1]
```diff
@@ -751,15 +727,13 @@ func predicates(additionalSelector string, callback func(bool)) (predicate.Predi
...
-	return &handler.EnqueueRequestsFromMapFunc{
-		ToRequests: handler.ToRequestsFunc(func(mo handler.MapObject) []controllerruntime.Request {
-			return []controllerruntime.Request{{NamespacedName: ctrlruntimeclient.ObjectKey{
-				Namespace: prowJobNamespace,
-				Name:      mo.Meta.GetName(),
-			}},
-			}
-		}),
-	}
+	// TODO alvaroaleman: This shouldn't be taking a MapObject but an Object
```
Comment (e-blackwelder) [inline prow/plank/reconciler.go:730]: It looks like you already made that change here. (So, is the TODO still applicable?)

**Review by e-blackwelder — APPROVED** [review]
/hold

Looks good to me, but waiting on the WIP (and a "nit" about your TODO comment).

File: prow/plank/reconciler.go (hunk 1 above)
Comment (alvaroaleman) [reply prow/plank/reconciler.go]: good catch, removed it

**Review by petr-muller — APPROVED** [review]

**Pull request merged**